compiler/src/dotty/tools/dotc/typer/Checking.scala

```diff
@@ -156,7 +156,7 @@ object Checking {
     if (arg.tpe.widen.isRef(defn.NothingClass) ||
         !paramBounds.exists ||
         arg.tpe.hasSameKindAs(paramBounds.bounds.hi)) arg
-    else errorTree(arg, em"Type argument ${arg.tpe} has not the same kind as its bound $paramBounds")
+    else errorTree(arg, em"Type argument ${arg.tpe} isn't the same kind as its bound $paramBounds")
```

Comment:
`does not have`. The English are doers. This has resulted in systemic inequality between the `haves` and the `have nots`. Gordias was a `has knot`.

Reply:
I accede.

Reply:
`is not of the same kindedness`!

"Seth is so kind, I'm not even of the same kindedness."